app/src/main/java/org/mozilla/focus/browser/cursor/CursorViewModel.kt

```diff
@@ -21,7 +21,7 @@ import java.util.concurrent.TimeUnit
 private const val UPDATE_DELAY_MILLIS = 17L // ~60 FPS.
 private const val UPDATE_DELAY_MILLIS_F = UPDATE_DELAY_MILLIS.toFloat() // Avoid conversion in update loop.
 
-private const val ACCEL_MODIFIER = 0.98f
+private const val DELTA_VEL_CHANGE_PER_FRAME = 0.98f
```

Comment:
This should still be accel, not vel - otherwise this would indicate that the cursor slows down over time.

Reply:
Going to switch it back to ACCEL_MODIFIER because `CHANGE_PER_FRAME` indicates this is the whole change per frame, as opposed to some constant that is used in an algorithm to change the velocity.